Src/BootCamp.Chapter/ComputerFactory.cs

```diff
@@ -0,0 +1,57 @@
+﻿using BootCamp.Chapter.Computer;
+using System;
+
+
+namespace BootCamp.Chapter
+{
+    public abstract class ComputerFactory
+    {
+        public virtual DesktopComputer Assemble()
+        {
+            var computer = new DesktopComputer();
+            TakeBody(computer);
+            PlaceMotherBoard(computer);
+            PlaceCpu(computer);
+            PlaceGpu(computer);
+            PlaceRam(computer);
+            Console.WriteLine("the ms computer is ready to be shipped");
+            return computer;
+
+        }
+
+        public virtual void PlaceRam(DesktopComputer computer)
```

Comment:
Leave those specific methods up to each factory implementation. The task didn't say that there is a default implementation of how each part is placed.

Reply:
oke, I thought because of the title of the challenge I needed to use virtual methods. Changed it to a abstract method